index.html

```diff
@@ -24,6 +42,35 @@ <h2>Add a new book</h2>
         <button type="submit" class="add-btn">Add</button>
       </form>
     </section>
+
+    <section class="dis-none section3">
+      <h1>Contact Information</h1>
+      <p>Do you have any questions or suggestions or you'd want to drop us a hello?</p>
+      <ul>
+        <li>Nemwel Boniface</li>
+        <li>
+          <ul>
+            <li>Mail at nemwelboniface@outlook.com</li>
+            <li>Number at my Number &#128522;</li>
+            <li>Address at Nairobi, Kenya</li>
+          </ul>
+        </li>
+      </ul>
+      <ul>
+        <li>Brahim Didi</li>
+        <li>
+          <ul>
+            <li>Mail at brahimdidi@outlook.com</li>
+            <li>Number at my Number &#128522;</li>
+            <li>Address at Nouakchott, Mauritania</li>
+          </ul>
+        </li>
+      </ul>
+    </section>
+
+    <footer class="footer">
+      <p>&copy;Nemwel~Brahim 2022 all right Reeserved.</p>
+    </footer>
```

Comment:
[OPTIONAL] There is an overlap between the content and the footer. I would recommend fixing that to improve your UI design:
 your design:
<img width="860" alt="image" src="https://user-images.githubusercontent.com/77190203/153402025-1aa67561-5412-47b3-992a-51faf187083d.png">
<img width="879" alt="image" src="https://user-images.githubusercontent.com/77190203/153402154-48155578-59fe-4e02-b569-958c018b85df.png">

Reply:
Thank you for this suggestion I will work on it. I believe it happens on smaller screens maybe?